### ISSUE TITLE
upper body logging issues

### ISSUE THREAD

**Issue by cdondrup** [strands-project/strands_perception_people#164]
The ubd logging node only provided 6 DB entries over a whole day. @lucasb-eyer might want to have a look why that might be. Also it hogs a lot of CPU.

**Comment by lucasb-eyer** [comment]
Thank you for checking. That's weird as I observed it working fine for a day in g4s. Is there a way for me to get remote-acces? (email?) Unfortunately I'm working on a paper deadline so don't have much time to check it.

It _also_ hogging the CPU might indicate some problem with queueing messages.

**Comment by cdondrup** [comment]
Has this been solved @lucasb-eyer ?

**Comment by lucasb-eyer** [comment]
Nope, not on my end. It (apparently) worked just fine during the review, so I still have no idea what may have caused this. Do you have that problem right, or are you triaging issues? If the latter, I'll let this sleep until I get back to working on it, since I (still) don't have much time for it.

**Comment by cdondrup** [comment]
The latter ;)
